src/Core/src/Layouts/LayoutExtensions.cs

```diff
@@ -188,5 +188,27 @@ public static Size AdjustForFill(this Size size, Rect bounds, IView view)
 
 			return size;
 		}
+
+		internal static Size ArrangeScrollViewContent(this IScrollView scrollView, Rect bounds) 
```

Comment:
Since we have to call this code from the stub and from controls would it make sense to move this code into the core handler itself? Set it up so the arrange call on each ScrollViewHandler just calls this method? This way future frameworks (comet, etc..) will also layout correctly?

Reply:
It shouldn't be determined by the handler, it should just be public (like `ComputeFrame()`, etc.) so other SDKs can use it if they want to. 
The stub really _shouldn't_ be calling it; that was just to make the layout more realistic for testing. It's really just a red flag that the test doesn't belong in Core, it belongs in Controls. I'm going to move thing around a bit so they make more sense.